cmd2/utils.py

```diff
@@ -644,3 +629,154 @@ def basic_complete(text: str, line: str, begidx: int, endidx: int, match_against
     :return: a list of possible tab completions
     """
     return [cur_match for cur_match in match_against if cur_match.startswith(text)]
+
+
+class TextAlignment(Enum):
+    LEFT = 1
+    CENTER = 2
+    RIGHT = 3
+
+
+def align_text(text: str, *, fill_char: str = ' ', width: Optional[int] = None, tab_width: int = 4,
+               alignment: TextAlignment) -> str:
```

Comment:
`alignment` appears to be a required keyword-only argument.  I would recommend doing one of the following:
* Make it a positional argument
* Move it to be the 1st keyword-only argument
* Give it a default value

Reply:
I'll make it positional